### ISSUE TITLE
Test suite `derive` failed to build on Stackage nightly

### ISSUE THREAD

**Issue by mihaimaruseac** [rudymatela/extrapolate#1]
```haskell
Building test suite 'derive' for extrapolate-0.2.0..
[1 of 1] Compiling Main             ( tests/test-derive.hs, dist/build/derive/derive-tmp/Main.o )

tests/test-derive.hs:5:1: error:
    Could not find module ‘Test’
    Use -v to see a list of the files searched for.
  |
5 | import Test
  | ^^^^^^^^^^^
```

**Comment by rudymatela** [comment]
I'm sorry about the failure.  Thanks for reporting the issue.

I'll look into this later today.

**Comment by rudymatela** [comment]
Turns out the `other-modules` field for some tests were missing on `extrapolate.cabal`.  Commit 	24307d2 fixes this.  I have added a test (565379a34b9ff73bd4da68b736d3d87f0d6c7314) so that I do not make this mistake again.  Sorry about my sloppiness.

I have released [Extrapolate v0.2.1] which should not have this problem.  [Extrapolate v0.2.1 builds and tests fine on Travis](https://travis-ci.org/rudymatela/extrapolate/builds/263145269).

When you have the time, you can revert fpco/stackage@1b741b0 too see if it works.

Again, thanks for the bug report.

[Extrapolate v0.2.1]: https://hackage.haskell.org/package/extrapolate-0.2.1

**Comment by mihaimaruseac** [comment]
Thank you. It's fixed, indeed.
